### ISSUE TITLE
The Napalm ring doesn't work.

### ISSUE THREAD

**Issue by PurpleManiac** [daleao/modular-overhaul#63]
Hello! I can't use the Napalm ring or fuse it with other rings.

![Captura de Tela (15)](https://github.com/daleao/modular-overhaul/assets/145059937/24afd15b-a59d-413a-897e-26da0fdb8ba0)

I would like to ask if it is possible to see any compatibility or possibility of finding out why the "Forge" option does not appear when we are using mods that change Clint's behavior, such as "Self Service" [https://www.nexusmods.com/stardewvalley/mods /11313](url) and "Clint Let Me Shop" [https://www.nexusmods.com/stardewvalley/mods /16738](url).

I believe that the questions about the FORGE function not appearing after the mission are because people use one or both of the above mods a lot.

Furthermore, I hope to be contributing to MARGO, it's simply incredible! A beautiful work.

**Comment by PurpleManiac** [comment]
Thanks for the answer, I'm using the most current version of MARGO and I'm having this problem with the Napalm ring as pointed out.

Here is my log:

https://smapi.io/log/58f83ac98b4048b3897b3e6fa7cb01b2

**Comment by daleao** [comment]
Can you confirm whether this is limited to Napalm ring, or all rings?

**Comment by PurpleManiac** [comment]
> Você pode confirmar se isso está limitado ao anel Napalm ou a todos os anéis?

Hello, unfortunately I reinstalled Windows on my PC and lost my save so I can't test the other rings as you asked me to.

But, I created a new save, and through the CJB item spawner, I checked all the rings and I can use them all, at least at the beginning of a new save.

I will do a new run and report if the problem persists, but I believe it may have been a specific problem on my machine, for example I did not have the net framework application updated. I'm happy that it seems like it was just a specific problem with my setup.
